app/controllers/api/v1/invoice_items/finds_invoice_items_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Api::V1::InvoiceItems::FindsInvoiceItemsController < ApplicationController
```

Comment:
🎉  I like that you have a custom find controller for invoice items. You could shorten the name to `Api::V1::InvoiceItems::FindController`. Since you are namespacing within InvoiceItems already, you don't need to add it again to the find part.